### PR TITLE
Fix internal shared_models package

### PR DESCRIPTION
**Pull request by nickmeinhold**
Fix shared models package

- get rid of old files causing errors 
- saved the generated files to version control

### CODE REVIEW

File: lib/widgets/auth/auth_page_buttons/stretchable_button.dart [hunk 1]
```diff
@@ -48,22 +48,22 @@ class StretchableButton extends StatelessWidget {
...
-              children: contents,
+            height: 40.0,
+            padding: EdgeInsets.all(buttonPadding ?? 2),
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(borderRadius),
+              side: bs,
             ),
-          ),
-        );
+            child: ElevatedButton(
```
Comment (nickmeinhold) [inline lib/widgets/auth/auth_page_buttons/stretchable_button.dart:57]: This is an unrelated change I was hoping to sneak in - swapping out the deprecated RaisedButton for the suggested ElevatedButton

**Review by T-Davis — APPROVED** [review]
Looks good, thanks for the comment on the button 👍
